src/Makefile

```
@@ -320,15 +320,44 @@ ifeq ($(BUILD_TLS),module)
 	TLS_MODULE_CFLAGS+=-DUSE_OPENSSL=$(BUILD_MODULE) $(OPENSSL_CFLAGS) -DBUILD_TLS_MODULE=$(BUILD_MODULE)
 endif
 
+define MAYBE_UNINSTALL_REDIS_SYMLINK
+    @if [ $(USE_REDIS_SYMLINKS) = "yes" ]; then \
+		rm -f $(1)/$(subst $(ENGINE_NAME),redis,$(2)); \
+    fi
+endef
+
 ifndef V
     define MAKE_INSTALL
         @printf '    %b %b\n' $(LINKCOLOR)INSTALL$(ENDCOLOR) $(BINCOLOR)$(1)$(ENDCOLOR) 1>&2
         @$(INSTALL) $(1) $(2)
     endef
+
+    define MAYBE_INSTALL_REDIS_SYMLINK
+        @if [ $(USE_REDIS_SYMLINKS) = "yes" ]; then \
+            $(call INSTALL_REDIS_SYMLINK,$(1),$(2)) \
+        fi
+    endef
+
+	define INSTALL_REDIS_SYMLINK
```

Indentation looks odd here. Is it a tab? (Tabs are necessary in Make only for the body of a make target. Otherwise, it's syntactically valid to use spaces.)

```suggestion
    define INSTALL_REDIS_SYMLINK
```